Makefile

```diff
@@ -20,9 +20,14 @@ run: ## Run via godep (without building)
 	go run *.go
 
 all: test build docker ## Test, build and docker image build
+	
+fmt: ## Run go fmt for all files in the project (excluding vendor)
+	go fmt $(shell go list ./... | grep -v /vendor/)
```

Comment:
I agree. Both options are valid and recommended, but are up to a developers discretion. I have my editor configured now for running gofmt automatically.

But because what a developer runs locally on their machine can't _100%_ be re-enforced, I prefer to also have it as a build step so it's guaranteed to be verified.